include/z64actor.h

```diff
@@ -146,12 +146,14 @@ typedef struct {
 #define ACTOR_FLAG_4 (1 << 4)
 #define ACTOR_FLAG_5 (1 << 5)
 #define ACTOR_FLAG_6 (1 << 6)
-#define ACTOR_FLAG_7 (1 << 7)
+// hidden or revealed by Lens of Truth (depending on room lensMode)
+#define ACTOR_FLAG_REACT_TO_LENS (1 << 7)
 #define ACTOR_FLAG_8 (1 << 8)
 #define ACTOR_FLAG_9 (1 << 9)
 #define ACTOR_FLAG_10 (1 << 10)
 #define ACTOR_FLAG_ENKUSA_CUT (1 << 11)
-#define ACTOR_FLAG_IGNORE_QUAKE (1 << 12) // actor will not shake when a quake occurs
+// Actor will not shake when a quake occurs
+#define ACTOR_FLAG_IGNORE_QUAKE (1 << 12)
```

Comment:
I personally prefer to keep the comments on the side when short enough (which most of them will be small enough to fit, as seen in my old flags pr)
for example 
```c
#define ACTOR_FLAG_REACT_TO_LENS (1 << 7) // hidden or revealed by Lens of Truth (depending on room lensMode)
```
If not, and we want to keep all on top, I need line breaks between flags for my eyes to easily see which comment belongs to what
like so
```c
// hidden or revealed by Lens of Truth (depending on room lensMode)
#define ACTOR_FLAG_REACT_TO_LENS (1 << 7)

// comment
#define ACTOR_FLAG_8 (1 << 8)

// comment
#define ACTOR_FLAG_9 (1 << 9)

// comment
#define ACTOR_FLAG_10 (1 << 10)

// comment
#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)

// Actor will not shake when a quake occurs
#define ACTOR_FLAG_IGNORE_QUAKE (1 << 12)
```

Reply:
(the only one thats really long and cant fit in a one liner so far is `ACTOR_FLAG_TALK` in another PR, and for that one I will also advocate for a line break either now or sometime after its merged)

Reply:
I took this style from MM's z64actor.h for sync purposes, personally I'm fine doing it differently, either same line or with more line breaks

Another option since these comments are doc comments anyway could be

```c
/**
 * Comment
 */
#define blah1
/**
 * Comment
 */
#define blah2
```

Reply:
think even with block comments id want line breaks between each flag.
Im fine with the current suggestion with line breaks added, but block comments are fine too if people want

Reply:
I think with the line breaks, everything else in this PR is good for me

Reply:
believe all we're waiting for on this is the line breaks between flags and its good to go

Reply:
went ahead and made this change